README.md

```diff
@@ -10,30 +10,33 @@ The basics enable many things. Read below.
     
     - House must be clean
     - Homework must be done
-    - Clothing must be ready for next day
+    - Clothing must be ready for next day (if you have clothing)
```

Comment:
not everybody has clothing ready

Reply:
clarification (hey I know its been a hot minute)